snapshot/lib/snapshot/test_command_generator_xcode_8.rb

```diff
@@ -24,7 +24,10 @@ def generate(device_type: nil, language: nil, locale: nil)
 
       def pipe(device_type, language, locale)
         log_path = xcodebuild_log_path(device_type: device_type, language: language, locale: locale)
-        return ["| tee #{log_path.shellescape} | xcpretty #{Snapshot.config[:xcpretty_args]}"]
+        pipe = "| tee #{log_path.shellescape} "
+        pipe << "| xcpretty #{Snapshot.config[:xcpretty_args]}"
+        pipe << "> /dev/null" if Snapshot.config[:suppress_xcode_output]
```

Comment:
Do we need a space before the `>`?

Reply:
I had initially written it with a space, and while testing I noticed it'd print out e.g. `…| xcpretty  > /dev/null` (double spaces), so I removed the leading space here.
But now, revisiting this, there's also the `xcpretty_args` case, which needs a space.

Do we care about the command style if it contains double spaces? It's just a mere styling issue cuz obv it works with multiple extra whitespaces :) If we do care about making the command look pretty, I gotta check how to write this neatly in ruby 😇

Reply:
```suggestion
        pipe << "| xcpretty #{Snapshot.config[:xcpretty_args]}"
        pipe << " > /dev/null" if Snapshot.config[:suppress_xcode_output]
```